src/quo2/components/navigation/page_nav.cljs

```diff
@@ -148,10 +149,20 @@
                   :flex-direction  :row
                   :justify-content :flex-end)}
    (let [last-icon-index (-> right-section-buttons count dec)]
-     (map-indexed (fn [index {:keys [icon on-press type style] :or {type :grey}}]
+     (map-indexed (fn [index
+                       {:keys [icon on-press type style icon-override-theme]
+                        :or   {type :grey}}]
                     ^{:key index}
-                    [rn/view {:style (merge {:margin-right (if (not= index last-icon-index) 8 0)} style)}
-                     [button/button {:on-press on-press :icon true :type type :size 32}
+                    [rn/view
+                     {:style (merge
```

Comment:
```
(assoc style :margin-right (if...))
```
I think reversing should be more slightly more performant, and I think it's a bit clearer

Reply:
Totally agree!
Done